index.php

```diff
@@ -81,7 +81,7 @@ function getLastRevision($distro) {
 
 To install the latest stable version:
 <pre>
-bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)"
+sudo bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)"
```

Comment:
https://github.com/opencollab/llvm-jenkins.debian.net/blob/master/llvm.sh#L66_L69

But I found this script requires root to run. Is this for security reason to avoid directly using sudo for downloaded script in documentation?

Reply:
Different reasons:
* sudo isn't the only option to install it 
* I expect people who needs these packages to know about sudo
* it is an unrelated change to adding an option for mirrors